### PR TITLE
[enterprise-3.5] oc describe node shows less information

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1572985

### CODE REVIEW

**Comment by sjenning** [comment]
LGTM

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

**Comment by mburke5678** [comment]
@zhouying7780 PTAL

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -78,33 +78,72 @@ $ oc describe node <node>
...
 ----
-====
+
+<1> The name of the node.
+<2> The role of the node, either `master` or `compute`.
+<3> The xref:../install_config/install/advanced_install.adoc#configuring-node-host-labels[labels] applied to the node.
+<4> xref:../admin_guide/manage_nodes.adoc#node-conditions[Node conditions].
+<5> The IP address and host name of the node. 
+<6> The xref:../admin_guide/allocating_node_resources.adoc#admin-guide-allocating-node-resources[pod resources and allocatable resources].
+<7> Information on the node host.
```
Comment (kalexand-rh) [inline admin_guide/manage_nodes.adoc]: s/on/about

File: admin_guide/manage_nodes.adoc [hunk 2]
```diff
@@ -78,33 +78,72 @@ $ oc describe node <node>
...
+<8> The fully-qualified domain name where the node can be reached.
+<9> Pods on the node.
```
Comment (kalexand-rh) [inline admin_guide/manage_nodes.adoc]: maybe s/Pods/The pods

**Review by kalexand-rh — COMMENTED** [review]
Minor nitpicks, but LGTM

**Pull request merged**